pontoon/contributors/tests/test_utils.py

```diff
@@ -27,7 +27,8 @@ def action_a(translation_a):
         action_type=ActionLog.ActionType.TRANSLATION_CREATED,
         translation=translation_a,
     )
-    action.created_at = datetime(2020, 1, 1)
+    aware_datetime = timezone.make_aware(datetime(2020, 1, 1))
+    action.created_at = aware_datetime
```

Comment:
Nit: I think it would be clearer if you put everything in one line:
```python
action.created_at = timezone.make_aware(datetime(2020, 1, 1))
```

Also below.